index.bs

```diff
@@ -543,6 +544,53 @@ In the schema definitions in this section, `<vendor>` is a placeholder for an <a
 
 Note: The prefix could be, for example, `nvda`, `jaws`, or `macOSVoiceOver`.
 
+A [=remote end=] has an associated set of <dfn for="remote end">supported settings</dfn>, which is either null or a [=set=] of strings which contains the name of every setting that may be referenced by this [=module=].
+
+<div algorithm>
+
+To determine if a <dfn>setting name is invalid</dfn> given string |name|:
+
+1. If [=supported settings=] is null:
+    1. Return false.
+2. If [=supported settings=] [=list/contains=] |name|:
```

Comment:
Should this not be the inverse? If `supported settings` does _not_ contain `name`?

Reply:
I believe it's correct as written. This algorithm is titled "setting name is invalid." If a given name is in the set of supported settings, then that name is *not* invalid, and the algorithm should return false.

I recognize that this design is something of a double-negative and that this can be confusing. It would certainly be more clear if the algorithm instead determined whether the input name were *valid*.

I've nonetheless proposed the more awkward version because it seems more reflective of implementation reality. As documented by the "Note" introduced in this patch, we'd like to support implementations which tolerate changing the value of "settings" which have no effect on the application's behavior. The first branch of this algorithm is intended to support those implementations. For that specific condition, it seemed misleading for the algorithm to report that a setting is "valid." It felt slightly more appropriate for it to report that the setting is "not invalid."

All that said, even if you agree with my motivation, it's probably too nuanced to justify the wording. I'm going to suggest an improvement with a follow-up patch since that would be an editorial enhancement, and I'd like to get the semantic change merged now.